src/Intl/Grapheme/Grapheme.php

```diff
@@ -177,7 +177,11 @@ private static function grapheme_position($s, $needle, $offset, $mode)
         if ($offset > 0) {
             $s = self::grapheme_substr($s, $offset);
         } elseif ($offset < 0) {
-            $offset = 0;
+            if (PHP_VERSION_ID < 50535 || PHP_VERSION_ID < 50621 || PHP_VERSION_ID < 70006) {
```

Comment:
this is wrong, as it will also match PHP 5.6.22 (as 50622 is smaller than 70006). It should be this:

```php
if (PHP_VERSION_ID < 50535 || (PHP_VERSION_ID >= 50600 && PHP_VERSION_ID < 50621) || (PHP_VERSION_ID >= 70000 && PHP_VERSION_ID < 70006)) {
    $offset = 0;
} else {
    return false;
}
```